mappings/net/minecraft/client/option/KeyBind.mapping

```diff
@@ -1,37 +1,38 @@
-CLASS net/minecraft/unmapped/C_sygikzty net/minecraft/client/option/KeyBinding
-	FIELD f_axzzilwx KEYS_BY_ID Ljava/util/Map;
+CLASS net/minecraft/unmapped/C_sygikzty net/minecraft/client/option/KeyBind
+	FIELD f_axzzilwx KEY_BINDS Ljava/util/Map;
 	FIELD f_dabvqpsd translationKey Ljava/lang/String;
 	FIELD f_dynqxsuy timesPressed I
-	FIELD f_frtwuquy CATEGORY_ORDER_MAP Ljava/util/Map;
+	FIELD f_frtwuquy ORDER_BY_CATEGORIES Ljava/util/Map;
 	FIELD f_isrvhtqu MOVEMENT_CATEGORY Ljava/lang/String;
 	FIELD f_kcphxfdy GAMEPLAY_CATEGORY Ljava/lang/String;
-	FIELD f_khpbnlde KEY_TO_BINDINGS Ljava/util/Map;
+	FIELD f_khpbnlde KEY_BINDS_BY_KEY Ljava/util/Map;
```

Comment:
`KEY_BINDS_BY_KEY_CODE`? the last `KEY` is vauge

Reply:
Key as in InputUtil's Key class, so, yeah, it's not only the key code